addons/l10n_sa_edi/models/account_journal.py

```diff
@@ -611,7 +611,7 @@ def _l10n_sa_call_api(self, request_data, request_url, method):
             # In the 401+ cases, it is like the server is overloaded e.g. and we still need to resend later.  We do not
             # erase the index chain (excepted) because for ZATCA, one ICV (index chain) needs to correspond to one invoice.
             status_code = ex.response.status_code
-            if status_code != 400:
+            if status_code not in (400, 409):
```

Comment:
Nitpick but the use of set in those cases is better :eyes: 
```suggestion
            if status_code not in {400, 409}:
```